example/index.js

```diff
@@ -1,14 +1,14 @@
 'use strict';
 const React = require('react');
 const ReactDOM = require('react-dom');
-const { Icon, Tooltip, Textfield, Switch, Spinner, Slider, Radio, Progress, IconToggle, Button, Badge, Checkbox } = require('../');
+const { TextFieldContainer, TextFieldInput, TextFieldError, TextFieldLabel, TextFieldTextArea, Icon, Tooltip, Switch, Spinner, Slider, Radio, Progress, IconToggle, Button, Badge, Checkbox } = require('../');
```

Comment:
TextFieldTextArea is destructured wrong (it is properly named TextFieldTextarea everywhere else)